.pyup.yml

```diff
@@ -0,0 +1,3 @@
+branch: master
+schedule: "everyweek on monday"
```

Comment:
Does this mean it will only send one pull request on Monday? Or will it still send *multiple* pull requests for each dependency on Monday? I'd like to only have one pull request.

Reply:
According to the docs it will create ONE PR, with all the updates. https://pyup.io/docs/bot/config/#schedules